pkg/k8sutil/k8sutil.go

```diff
@@ -198,9 +198,6 @@ func GetMinorVersion(dclient discovery.DiscoveryInterface) (int, error) {
 // the associated yaml asset
 func NewCustomResourceDefinition(crdKind monitoringv1.CrdKind, group string, labels map[string]string, validation bool) *extensionsobj.CustomResourceDefinition {
 	crdName := strings.ToLower(crdKind.Plural)
-	if crdName == "prometheuses" {
-		crdName = "prometheus"
-	}
 	assetPath := "example/prometheus-operator-crd/" + group + "_" + crdName + ".yaml"
 	data := monitoringv1.MustAsset(assetPath)
```

Comment:
can we iterate in an unit test over all embedded assets and invoke `monitoringv1.MustAsset` so we have confidence this actually works?

Otherwise this would be invoked just at runtime. I'd maybe even add the 
```
crd := &extensionsobj.CustomResourceDefinition{}
err := yaml.Unmarshal(data, crd)
```

in the unit test.

Reply:
@s-urbaniak I added a unit test that calls `NewCustomResourceDefinition` for each of the crd types.  Do you think this is sufficient, or should we also call Unmarshal directly in some cases?